autoload/dracula.vim

```diff
@@ -54,4 +54,13 @@ func! dracula#should_abort(...)
     return 0
 endfunction
 
+" Use termguicolors instead of 256
+if (has('nvim'))
+  let $NVIM_TUI_ENABLE_TRUE_COLOR = 1
```

Comment:
I have no idea what this does--is it neovim's version of `termguicolors`?

Does neovim ubiquitously support truecolor, even when running in terminals without it?

Reply:
FYI:

![image](https://user-images.githubusercontent.com/5240018/126638547-d182b5d9-8eea-4cc1-ac85-253e9610c8af.png)

https://twitter.com/justinmk/status/730614264659120128

Reply:
Ok, so that would need fixed anyhow.